cloud/user/user.go

```diff
@@ -1,26 +1,29 @@
 package user
 
 import (
+	http_context "context"
```

Comment:
```suggestion
	httpContext "context"
```
in order to keep the naming convention consistent